docs/content/how-to/openstack/additional-ports.md

```diff
@@ -0,0 +1,50 @@
+# Additional ports for Nodepools
+
+This document describes how to attach additional ports to nodes in a HostedCluster on OpenStack.
+
+## Use-cases
+
+- **SR-IOV**: Single Root I/O Virtualization (SR-IOV) is a specification that allows a single Peripheral Component Interconnect Express (PCIe) physical device to appear as multiple separate physical devices. This can be useful for high-performance networking scenarios. The Nodepool can be configured to use SR-IOV by attaching additional ports to the nodes. Workloads that require high-performance networking can use the SR-IOV devices.
+- **DPDK**: Data Plane Development Kit (DPDK) is a set of libraries and drivers for fast packet processing. DPDK can be used to improve the performance of network applications. The Nodepool can be configured to use DPDK by attaching additional ports to the nodes. Workloads that require high-performance networking can use the DPDK devices.
+- **Manila RWX on NFS**: Manila ReadWriteMany (RWX) volumes can be used by multiple nodes in a HostedCluster. The Nodepool can be configured to use Manila RWX volumes by attaching additional ports to the nodes. Workloads that require shared storage will be able to reach the NFS network configured for the Manila RWX volumes.
+- **Multus CNI**: Multus is a meta-plugin for Kubernetes CNI that enables attaching multiple network interfaces to pods. The Nodepool can be configured to use Multus by attaching additional ports to the nodes. Workloads that require multiple network interfaces will be able to use the Multus CNI. For example it's useful if the workload needs to connect to IPv6 networks (dual-stack or single-stack).
+
+## Prerequisite
+
+Additional networks must be created in OpenStack and the project used by the HostedCluster must have access to these networks.
+
+## Available options
+
+The `--openstack-node-additional-port` flag can be used to attach additional ports to nodes in a HostedCluster on OpenStack. The flag takes a list of parameters separated by commas. The parameter can be used multiple times to attach multiple additional ports to the nodes.
```

Comment:
Vaguely related: this isn't a great name for that option since it suggests a port ID (which makes no sense since that would be limited to one instance).

Reply:
I wonder if it's too late for an API / CLI change, please propose something that you think would be better. I guess it's fine since we are dev preview, however the API is v1beta1.
However one can argue that "additional-port" makes sense because we define how the port should be configured: which network ID it belongs, what vnic type, whether port security is enabled and allowed address pairs. It's port options, not just network, in my opinion.
Either way, I'll be fine if you propose a change, before we have real customers.